lib/blacklight/solr_response/pagination_methods.rb

```diff
@@ -14,21 +14,4 @@ def offset_value #:nodoc:
   def total_count #:nodoc:
     total
   end
-
-  def model_name
```

Comment:
`#model_name` didn't seem to work (or, when it worked like I would expect, we had test failures.), so I've removed it.